lab-max/app/scss/lib/_theme.scss

```diff
@@ -0,0 +1,9 @@
+$app-primary: #f7aa7f;
+$app-secondary: #FFF930;
+$app-tertiary: #B36382;
+$app-success: #84FF30;
+$app-active: #8FFF96;
+$app-disabled: #F00;
+$app-error: #B32031;
+
+$app-font: Helvetica;
```

Comment:
@meeksfred I usually create a font stack, for old browser fallbacks, when determining a font - you can google for additional info